Python/Setup/PythonTools/PythonToolsFiles.proj

```diff
@@ -135,7 +136,11 @@
     <File Include="$(BuildOutputRoot)Binaries\Analyzer\CompletionDB\*.idb">
       <InstallDirectory>CompletionDB</InstallDirectory>
     </File>
-  
+
+    <File Include="*.idb">
+      <InstallDirectory>CompletionDB</InstallDirectory>
+    </File>
+    

```

Comment:
This section looks like an incorrect merge, as it should be covered by the File element above it.